tutorials/plot_receptive_field.py

```diff
@@ -259,4 +264,82 @@
 fig.suptitle('Model coefficients / scores for many ridge parameters', y=1)
 mne.viz.tight_layout()
 
+###############################################################################
+# Using different regularization types
+# ------------------------------------
+# In addition to the standard ridge regularization, the
+# :class:`mne.decoding.TimeDelayingRidge` class also exposes quadratic
+# regularization term as:
+#
+# .. math::
+#    \left[\begin{matrix}
+#         1 & -1 &   &   & & \\
+#        -1 &  2 & -1 &   & & \\
+#           & -1 & 2 & -1 & & \\
+#           & & \ddots & \ddots & \ddots & \\
+#           & & & -1 & 2 & -1 \\
+#           & & &    & -1 & 1\end{matrix}\right]
+#
+# This imposes a smoothness constraint of nearby time samples. Quoting [5]_:
+#
+#    Tikhonov [identity] regularization (Equation 5) reduces overfitting by
+#    smoothing the TRF estimate in a way that is insensitive to
+#    the amplitude of the signal of interest. However, the quadratic
+#    approach (Equation 6) reduces off-sample error whilst preserving
+#    signal amplitude (Lalor et al., 2006). As a result, this approach
+#    usually leads to an improved estimate of the system’s response (as
+#    indexed by MSE) compared to Tikhonov regularization.
+#
+
+alphas = np.logspace(-4, 0, 10)
+old_scores = scores
+scores = np.zeros_like(alphas)
+models = []
+for ii, alpha in enumerate(alphas):
+    estimator = TimeDelayingRidge(tmin, tmax, sfreq, reg_type='quadratic',
```

Comment:
did you always look at the validity of the results when using ['quadratic', 'ridge'] or vice versa?